class  C2
{
	int test()
	{
	try
	{
		//several statement
	}
	catch (ArithmeticException ex)
	{
				return 10;
	}
	finally
	{
	
	}
	return 40;
	}
}
